test_integration/contracts/EIP20Token.json

```diff
@@ -0,0 +1,243 @@
50905090810190601f1680156101195780820380516001836020036101000a031916815260200191505b509250505060405180910390f35b34801561013357600080fd5b506101606004803603604081101561014a57600080fd5b50600160a060020a03813516906020013561035b565b604080519115158252519081900360200190f35b34801561018057600080fd5b506101896103c1565b60408051918252519081900360200190f35b3480156101a757600080fd5b50610160600480360360608110156101be57600080fd5b50600160a060020a038135811691602081013590911690604001356103c7565b3480156101ea57600080fd5b506101f3610432565b6040805160ff9092168252519081900360200190f35b34801561021557600080fd5b506101896004803603602081101561022c57600080fd5b5035600160a060020a031661043b565b34801561024857600080fd5b506100b2610456565b34801561025d57600080fd5b506101606004803603604081101561027457600080fd5b50600160a060020a0381351690602001356104b6565b34801561029657600080fd5b50610189600480360360408110156102ad57600080fd5b50600160a060020a03813581169160200135166104ca565b60008054604080516020601f60026000196101006001881615020190951694909404938401819004810282018101909252828152606093909290918301828280156103515780601f1061032657610100808354040283529160200191610351565b820191906000526020600020905b81548152906001019060200180831161033457829003601f168201915b5050505050905090565b336000818152600560209081526040808320600160a060020a038716808552908352818420869055815186815291519394909390927f8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925928290030190a350600192915050565b60035490565b600160a060020a03831660009081526005602090815260408083203384529091528120546103fb908363ffffffff6104f516565b600160a060020a038516600090815260056020908152604080832033845290915290205561042a84848461056c565b949350505050565b60025460ff1690565b600160a060020a031660009081526004602052604090205490565b60018054604080516020601f600260001961010087891615020190951694909404938401819004810282018101909252828152606093909290918301828280156103515780601f1061032657610100808354040283529160200191610351565b60006104c333848461056c565b9392505050565b600160a060020a03918216600090815260056020908152604080832093909416825291909152205490565b60008282111561056657604080517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152601b60248201527f556e646572666c6f77207768656e207375627472616374696e672e0000000000604482015290519081900360640190fd5b50900390565b600160a060020a038316600090815260046020526040812054610595908363ffffffff6104f516565b600160a060020a0380861660009081526004602052604080822093909355908516815220546105ca908363ffffffff61062b16565b600160a060020a0380851660008181526004602090815260409182902094909455805186815290519193928816927fddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef92918290030190a35060019392505050565b6000828201838110156104c357604080517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152601560248201527f4f766572666c6f77207768656e20616464696e672e0000000000000000000000604482015290519081900360640190fdfea165627a7a723058205d93888008fa3996759fca88e66232a119e141917a86569af7e5cd7b6a096cb40029",
```

Comment:
Why is this filed added? We can get the abi and bin from the npm?

Reply:
I am assuming a that this is a "real" EIP20Token, in contrast to the one we are currently exposing from mosaic-contracts. See also https://github.com/OpenSTFoundation/mosaic-contracts/issues/629

Reply:
It is. It is not part of `mosaic-contracts`.